pkg/asset/ignition/bootstrap/baremetal/template_test.go

```diff
@@ -59,6 +59,7 @@ func TestTemplatingManagedIPv6(t *testing.T) {
 		ProvisioningDHCPRange:   "fd2e:6f44:5dd8:b856::1,fd2e:6f44:5dd8::ff",
 		BootstrapProvisioningIP: "fd2e:6f44:5dd8:b856::2",
 		ProvisioningNetwork:     baremetal.ManagedProvisioningNetwork,
+		APIVIPs:                 []string{"53.78.144.26", "d601:602e:6397:a048:f516:dc63:1e83:fcaa"},
```

Comment:
Shouldn't the APIVIPs contain only IPv6 address since this is not dualstack test ?

Reply:
I see, thanks for clarifying :)